### PR TITLE
Point to existing install instructions

### PR DESCRIPTION
**Pull request by mkcor**
Hello @plotly/dash @rmarren1 and @T4rk1n!

I'm trying to document the installation of dash-bio, so I came here for inspiration! In the current version of the README, there are no installation (let alone development) instructions. That's why I'm suggesting pointing to the user guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Getting Started
 
-Refer to the [readme](README.md) for installation and development instructions.
+Refer to the Dash User Guide for [installation](https://dash.plot.ly/installation) instructions.
```
Comment (T4rk1n) [inline CONTRIBUTING.md]: The installation instructions on the dash docs is for developing dash apps. 
Installation for development on the dash repo should be:

- Create a virtualenv: 
`python -m venv venv` or `virtualenv venv` for python 2.
- Activate the virtualenv:
`. venv/bin/activate` or `venv\scripts\activate` for windows.
- Install the dev dependencies:
`pip install -r .circleci/requirements/dev-requirements.txt`

Comment (mkcor) [reply CONTRIBUTING.md]: Thanks for reviewing, @T4rk1n! In light of your comment, I have submitted another change, which is more to the point. I don't think it is healthy to promote the use of Python 2 any more (I'll share references if you ask for them), so I have only included the Python 3 way.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
 
 ## Getting Started
 
-Refer to the [readme](README.md) for installation and development instructions.
+To set up your development environment, run the following commands:
+```bash
+# Create a virtualenv (name it, say, dash-dev)
+python3 -m venv dash-dev
+# Activate this virtualenv
+. dash-dev/bin/activate
```
[the remark this answers is not in the document]
Comment (mkcor) [reply CONTRIBUTING.md]: Good catch! I can see `venv` in the `.gitignore` file; I wasn't aware of this convention; I used to keep all my virtualenvs under `~/.virtualenv/` (calling for prior command `mkdir -p ~/.virtualenv/` as in http://nipy.org/nipy/devel/tools/virtualenv-tutor.html).

Lately, I have used almost only conda environments, following the [wave](https://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/) in the SciPy/PyData community, so thanks for the refresher! I'll make and push the change right now.

[the remark this answers is not in the document]
Comment (mkcor) [reply CONTRIBUTING.md]: @T4rk1n hmm, in the general case, newcomers willing to contribute would have to fork the repo first... I'll replace the first command with a generic comment.

[the remark this answers is not in the document]
Comment (mkcor) [reply CONTRIBUTING.md]: Not to mention that some people (including me) would clone over SSH ;)

**Comment by mkcor** [comment]
Voilà, @T4rk1n! I have used your convention of prefixing commands with `$ ` :)

**Comment by T4rk1n** [comment]
Looking good, thanks :dancer:

**Pull request merged**